guides/source/initialization.md

```diff
@@ -76,20 +76,27 @@ A standard Rails application depends on several gems, specifically:
 * activesupport
 * actionmailbox
 * actiontext
-* arel
+* bootsnap
 * builder
 * bundler
 * erubi
+* importmap-rails
 * i18n
+* jbuilder
 * mail
 * mime-types
+* puma
 * rack
 * rack-test
 * rails
 * railties
 * rake
+* redis
+* sprockets-rails
```

Comment:
Presumably `sprockets-rails` can be removed based on
https://github.com/rails/rails/blob/de7c495209811f8f918fa9f915e64df61a6080c1/guides/source/7_0_release_notes.md?plain=1#L39-L42

Reply:
I believe the `rails` gem doesn't depend on `sprockets-rails`, but when generating a new rails application via the command `rails new app_name`, `sprockets-rails` is added by default. So a standard `rails application` as opposed to the `rails gem` does depend on `sprockets-rails` right?

Reply:
I think you are right! 

https://github.com/rails/rails/blob/c9c1768a91c2b6e6ca68268ed87b3a7327ab81dd/railties/test/generators/plugin_generator_test.rb#L647-L652

I think it's reasonable to mention the gem as long as it stays in the default config
https://github.com/rails/rails/blob/c9c1768a91c2b6e6ca68268ed87b3a7327ab81dd/railties/lib/rails/generators/app_base.rb#L63

Sorry for the confusion